databox/client/connections/DisconnectedFromCoreError.ts

```diff
@@ -0,0 +1,12 @@
+import DisconnectedError from '@ulixee/net/errors/DisconnectedError';
```

Comment:
This class is identical to the one in net. Seems unnecessary?

Reply:
It's not exactly the same. It modifies the error message to be specific to Databox. 

Hero as the same class defined in it's connections folder, which is where I took it from.

Reply:
I think we need to at minimum change the "name" property to be unique here or TypeSerializer will instantiate whichever one registers last when it tries to rehydrate. But I find this of limited value since you can just create a DisconnectedError with a custom message...

Reply:
Actually, from what I can tell, this is only being used in commented-out code..

Reply:
It's being used in DataboxInternal code which is not commented out.

Reply:
Are you suggesting we remove it in Hero as well? I was trying to keep the two in sync as much as possible.

Reply:
We are checking for it being thrown, but it's never being thrown from what I can see. Hero has connect and disconnect messages that it sends across the wire. They're a source of a good bit of complication, but I don't know how to remove them from that side. This code seems to have been partially copied, but I'm not sure it's doing anything.

Reply:
Ha. Ok. Will remove.

Reply:
Removed.